spanner/sessionclient.go

```diff
@@ -274,6 +274,8 @@ func (sc *sessionClient) executeBatchCreateSessions(client spannerClient, create
 			break
 		}
 		var mdForGFELatency metadata.MD
+		// Each invocation of client.BatchCreateSessions is not automatically retried
```

Comment:
Same question here as for PartitionQuery; I don't quite understand what you mean with 'not automatically retried' in these cases. Could you elaborate a bit on that?